Applications/ApplicationsLib/ProjectData.h

```diff
@@ -63,7 +64,8 @@ class ProjectData final
 	/// The additional  path is used to find files referenced in the
 	/// configuration.
 	ProjectData(BaseLib::ConfigTree const& config_tree,
-	            std::string const& path);
+	            std::string const& path,
+	            std::string const& output_directory);
```

Comment:
I prefer the suffix `_directory`, because the arguments refer to directories.
From `..._path` it is not directly visible if files or dirs are meant.